SearchEngines.yml

```diff
@@ -8,6 +8,14 @@
     backlink: 's/{k}'
     charsets:
       - iso-8859-2
+118 700:
+  - 
+    urls:
+      - www.118700.se
+      - foretag.118700.se
```

Comment:
webben.118700.se  would be the most relevant here, as we primary have web search engines listed. could you please add it